bin/setup

```diff
@@ -48,6 +48,13 @@ else
   cp .env.example .env
 fi
 
+if test -f ".env.development"; then
+  echo "Found .env.development, leaving it in place!"
+else
+  echo "Copying .env.development.example to .env.development"
+  cp .env.development.example .env
+fi
+
```

Comment:
I have a feeling the `.devcontainer/startup.bash` script may need to be updated with this new `.env` changes?
Specifically, the block starting at [line 92](https://github.com/zinc-collective/convene/blob/51ff6f302ac669370e4636e8f3a6baa22a9c13fe/.devcontainer/startup.bash#L92C3-L92) may need to be adjusted.
This assumption may be wrong, but I wanted to bring it up sooner rather than later.

Reply:
It would be nice if the three places we use (bin/setup, bin/setup-rails, and .devcontainer/startup.bash) all used the same shell script to create these default ffiles; perhaps `bin/setup-env`? Are you available to do that refactor?

Reply:
Sure, I can do that.

One script to do a conditionally copy of both `.env.example` and `.env.development.example`?
And then in the `.devcontainer/startup.bash` script, do the `sed` stuff after running this script?
Mostly just probing to make sure I'm understanding what needs to be here.

Reply:
Yep! I think the sed stuff should stay in the startup.bash